tests/x11/ibus/ibus_test_jp.pm

```diff
@@ -0,0 +1,72 @@
+# SUSE's openQA tests
+#
+# Author: Gao Zhiyuan <zgao@suse.com>
+# Copyright © SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: setup and test ibus japanese input
+# Maintainer: Gao Zhiyuan <zgao@suse.com>
+
+use base "x11test";
+use strict;
+use testapi;
+use utils;
+
+sub ibus_enable_source_jp {
+    send_key 'super';
+    wait_still_screen;
+    type_string_slow ' region & language';
+    wait_still_screen;
+    send_key 'ret';
+    wait_still_screen;
```

Comment:
This is not optimal way of syncing the test, as we have clock and blinking cursor, so it becomes same as sleeping for 7 seconds.

Reply:
@rwx788  
Thank you for the comment!
Do you have a suggested replacement please? 
I also struggled a little bit with this part. 
A proper synchronization is truly needed because of [tw performance issues](https://bugzilla.opensuse.org/show_bug.cgi?id=1112824) and [typing problems via vnc](https://bugs.launchpad.net/qemu/+bug/1802465). 
I might want to use `wait_still_screen 3;` as a replacement, what do think?

Reply:
Ideally, with assert_screen calls, that's most transparent and stable way. As when we have multiple key presses and test fails because some was lost, without intermediate steps it's hard to understand what actually has happened. If this doesn't help, we could lower similarity level, so that blinking cursor and changing  clock doesn't affect it + decreasing still time to 2 or 3 seconds (do not confuse it with timeout), please, take a look here: http://open.qa/api/testapi/#_wait_still_screen

Reply:
Thank you very much! I removed `wait_still_screen` before `assert_screen`, and I reduced some of the `wait_still_screen` time to 3 and it seems to work!
I will update the branch and attach my new verification run very soon.